libraries/mapping/publish_form_mapper.php

```diff
@@ -659,6 +659,8 @@ private function get_video_streaming($asset, $profile): ?array
         ];
 
         if ($profile === 'player-video') {
+            $poster = '';
+            $video_url = $asset_current->enclosures[0]->href;
```

Comment:
Check your variable names. I'm showing $asset_current undefined at this location